src/Model.elm

```diff
@@ -221,16 +213,53 @@ update msg model =
                     ( model, Cmd.none )
 
 
-v2Clamp : V2.Vec2 -> V2.Vec2 -> V2.Vec2 -> Float -> V2.Vec2
-v2Clamp min max orig zoom =
+handleDrag : HomeModel -> V2.Vec2 -> Float -> V2.Vec2
+handleDrag home delta zoom =
+    let
+        g =
+            home.graph
+
+        --Similar to iconSize in ViewGraph (can't access it here)
```

Comment:
Moving it from ViewGraph to Model would be fine. Better, actually - here it's easy to update one and forget the other